analysed-packages/jquery-mousewheel/version-3.1.13/jquery-mousewheel-3.1.13-SPDX2TV.spdx

```diff
@@ -0,0 +1,389 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip_1713540928.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Parian Golchin
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of jquery-mousewheel 3.1.13.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-04-19T15:35:28Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip
+PackageFileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip
+SPDXID: SPDXRef-upload106
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 0fecaa8aa7ebee07908d21cde0414ed4b9ba65e2
+PackageChecksum: SHA1: 89e23279e56cf37cb293dbbb6b6673faa61d7d20
+PackageChecksum: SHA256: e77bae24414e5832cffef1ad5f33d7ae76e2976147900d2db5440ab2d71443b8
+PackageChecksum: MD5: 1006feb3bf96f458cc36ae1c85001b1c
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-MIT
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - reportImport (.)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload106
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/test/browserify/bundle.js
+SPDXID: SPDXRef-item460144
+FileChecksum: SHA1: defd6266a1032e042d9362568b8731019ad0f190
+FileChecksum: SHA256: 4f064e87ba28c9e0b98b4f3417a51fd14a66c545f42f2ac5f76a39d602ff87e0
+FileChecksum: MD5: 777ab9631946a7e8e094ece2134a0e5b
+LicenseConcluded: LicenseRef-MIT
+LicenseComments: <text> The information in the file is: Released under the MIT license http://jquery.org/license We followed the link and there it said: Projects referencing this document are released under the terms of the [MIT license](https://tldrlegal.com/license/mit-license). Thus, we concluded MIT License. </text>
```

Comment:
The License comment is not correct: the information in this file is:

>  * Licensed under the MIT License (LICENSE.txt).
 So the comment needs to be adapted